bundles/org.openhab.binding.siemensrds/README.md

```diff
@@ -0,0 +1,29 @@
+# Siemens RDS Binding
+
+The Siemens RDS binding allows you to connect openHab to Siemens Climtix IC cloud server and integrate your RDS smart thermostats onto the bus.
+
+## Binding Configuration
```

Comment:
I did use the official template, so I am surprised by your comment.
=> Can you please be more specific about what you don't like?

Reply:
Just a few points:
- What you have in "Binding Configuration" belongs into "Thing Configuration". 
- A section "Item Configuration" does not exist in the template.
- The "Channels" section is missing - this is the most important one (probably what you have as Items right now).
- Every README must have the "Full Example" section.

Reply:
I've done everything except the "Full Example" -- as there is nothing to show an example of. Or do you want (say) a PaperUI screenshot ?

Reply:
Please have a look at other bindings. A "Full Example" shows an example of a thing-file, an item-file and a sitemap-file that fit together.

Reply:
Hmm. I have been testing the binding using OH2 PaperUI, so I have not yet tried to create manual Thing/Item/Sitemap files.

I am reluctant to present example Thing files. I guess the Cloud Account Thing could be created by a file (e.g. putting the login credentials in a file) but I am pretty sure that Smart Thermostat Things can (must) only be created by Automatic Discovery. This is because Auto Discovery finds all Smart Thermostats according to their unique Plant Id codes in the cloud server account, and if the user would try to manually create such a Thing, they have no way to know what Plant Ids are existing in the account, and if they were to just guess a Plant Id that is not in the account, it would fail).

On the other hand, I can more or less understand the need for manual Item an Sitemap files, so I could probably create some examples of that.

Reply:
Every binding must support textual configuration and the examples are crucial for it.
If there is no way for the user to find out the PlantId on his account, he can still use discovery once for that purpose and then go on and configure the Things in a text file.

Reply:
Ok. I have added a Full Example to the ReadMe file. And while working on that I realised that that some of the naming of Channels could be improved for easier understanding, so I also changed the respective Java and XML files to match.